WorkspaceServer/WorkspaceFeatures/PackageTool.cs

```diff
@@ -28,11 +28,11 @@ public PackageTool(string name, DirectoryInfo workingDirectory)
 
         public string Name { get; }
 
-        public async Task<DirectoryInfo> LocateBuildAsset()
+        public async Task<ProjectAsset> LocateProjectAsset()
         {
             var result = await CommandLine.Execute(_path.Value, MLS.PackageTool.PackageToolConstants.LocateBuildAsset, _workingDirectory);
             var projectDirectory = new DirectoryInfo(string.Join("", result.Output));
-            return projectDirectory;
+            return new ProjectAsset(new FileSystemDirectoryAccessor(projectDirectory));
```

Comment:
>new FileSystemDirectoryAccessor(projectDirectory) [](start = 36, length = 49)

Similar issue. We should inject IDirectoryAccessor and move in the direction of in-memory testing for these things.